app/views/members/_member_page_header.html.haml

```diff
@@ -13,17 +13,30 @@
           %p.member-data.object-data
             - if member.person.rebellions_fraction
               %span.member-rebellions.object-data-rebellion
-                -if member.person.rebellions_fraction == 0
-                  Never rebels
-                -else
-                  -# TODO: Should this be an absolute count rather than percentage?
-                  -# Maybe it's good to show it as a percentage because it highlights rarity?
-                  Rebels
-                  = fraction_to_percentage_display(member.person.rebellions_fraction)
-                  of the time
-                  -# TODO: add helper tooltip for rebellions
-                  -# link_to "explain...", help_faq_path(anchor: "clarify")
+                - if !member.currently_in_parliament?
```

Comment:
@mario-areias I have made the changes as you suggested. Could you please take a look?